addons/knowledge/views/knowledge_views.xml

```diff
@@ -0,0 +1,247 @@
+                                                <div class="dropdown-divider"
+                                                    attrs="{'invisible': ['&amp;', ('last_edition_uid', '=', False), ('last_edition_date', '=', False)]}"/>
+                                                <div class="px-4 py-2 mx-1" attrs="{'invisible': [('last_edition_uid', '=', False)]}">
+                                                    <h6 class="small font-weight-bold m-0">Last Edited by:</h6>
+                                                    <div class="mb-2 text-nowrap">
+                                                        <field name="last_edition_uid" readonly="1"/> -
+                                                        <field name="last_edition_date" widget="datetime" readonly="1"/>
+                                                    </div>
+                                                    <h6 class="small font-weight-bold m-0">Created by:</h6>
+                                                    <div class="text-nowrap">
+                                                        <field name="create_uid" readonly="1"/> - 
+                                                        <field name="create_date" widget="datetime" readonly="1"/>
+                                                    </div>
+                                                </div>
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                            <div class="flex-grow-1 position-relative">
+                                <div class="o_scroll_view">
+                                    <!-- Article body -->
+                                    <div class="o_knowledge_editor d-flex border-bottom bg-white">
+                                        <field name="body" widget="html" no_label="1" default_focus="1"
+                                            options="{'resizable': False}"
+                                            attrs="{'readonly': ['|', ('is_locked', '=', True), ('user_can_write', '=', False)]}"/>
+                                    </div>
+                                    <!-- Chatter -->
+                                    <div class="o_knowledge_chatter_container">
+                                        <div class="oe_chatter">
+                                            <field name="message_follower_ids" groups="base.group_user"/>
+                                            <field name="message_ids"/>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="o_knowledge_resizer d-print-none d-none d-sm-block px-3 opacity-75 opacity-100-hover">
```

Comment:
Again, all those classes should probably be prefixed: "o_knowledge_article_form_resizer".

Good namespacing helps understanding where it's used, "o_knowledge" is a bit too vague.